miniflare/miniflare/extension.py

```diff
@@ -134,5 +138,4 @@ def _install(self, target: InstallTarget) -> None:
             run(["apt", "install", "-y", "libc++-dev"])
 
         # install npm package
-        run(["npm", "install", "--prefix", target_dir, "wrangler"])
-        run(["npm", "install", "--prefix", target_dir, "@miniflare/tre"])
```

Comment:
No longer required to install `miniflare` separately - now fully integrated into the `wrangler` CLI